org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/settings/SettingsTest.java

```diff
@@ -24,43 +29,56 @@
  */
 public class SettingsTest {
 
+	private static String testFolder = "TestXMLCacheFolder1122333";
+	private static String targetTestFolder = "target/generated-test-sources";
+	
+	
+	@After
+	public void cleanup() {
+		String path = System.getProperty("user.dir") + "/" + targetTestFolder + "/" + testFolder;

```

Comment:
you should not have to touch anything under user.dir

Reply:
This is for the test folder that I created, within the target folder.